ui/app/AppLayouts/stores/RootStore.qml

```diff
@@ -165,8 +172,8 @@ QtObject {
         return walletSectionTransactions.getChainIdForBrowser()
     }
 
-    function suggestedRoutes(account, amount, token, disabledChainIds) {
-        return JSON.parse(walletSectionTransactions.suggestedRoutes(account, amount, token, disabledChainIds)).networks
+    function suggestedRoutes(account, amount, token, disabledFromChainIDs, disabledToChainIDs, preferredChainIds, priority) {
```

Comment:
what do u think of having a transaction store? or send store. I believe this component is so complex it deserves his own store

That way we can remove the need to have sticker store injected into ens view?